core/src/main/java/com/linecorp/armeria/internal/annotation/AnnotatedBeanFactory.java

```diff
@@ -141,17 +166,17 @@ static synchronized BeanFactoryId register(Class<?> clazz, Set<String> pathParam
                         constructor.getValue(), resolverContext);
                 final T instance = constructor.getKey().newInstance(constructorArgs);
 
-                for (final Entry<Field, AnnotatedValueResolver> field : fields) {
-                    final Object fieldArg = field.getValue().resolve(resolverContext);
-                    field.getKey().set(instance, fieldArg);
-                }
-
-                for (final Entry<Method, List<AnnotatedValueResolver>> method : methods) {
+                for (final Entry<Method, List<AnnotatedValueResolver>> method : methods.entrySet()) {
```

Comment:
Could you add some comment about which is injected first? e.g. constructor, method, field

Reply:
I added comments before `findMethods` and `findFields` that the method will warn and not include if there're parameters used redundantly.